<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadRosterDialog</class>
 <widget class="QDialog" name="UploadRosterDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="chooseFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose File..</string>
   </property>
  </widget>
  <widget class="QLabel" name="menuTitle">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size: 30px;</string>
   </property>
   <property name="text">
    <string>SwapShift</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>File: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="uplRostOKButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="isValidRostLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="rostTitleLabel">
      <property name="text">
       <string>Roster Title (Eg. Week 1)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="rostTitleLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
